src/System.Private.ServiceModel/tests/Scenarios/Binding/Tcp/NetTcpBindingTests.cs

```diff
@@ -12,15 +12,28 @@ public partial class Binding_Tcp_NetTcpBindingTests : ConditionalWcfTest
     // Default settings are:
     //                         - SecurityMode = Transport
     //                         - ClientCredentialType = Windows
+#if FULLXUNIT_NOTSUPPORTED
+    [Fact]
+    [ActiveIssue(832)] // Windows Stream Security is not supported in NET Native
+#else
     [ConditionalFact(nameof(Root_Certificate_Installed), nameof(Windows_Authentication_Available))]
-#if !FEATURE_NETNATIVE
     [ActiveIssue(945, PlatformID.AnyUnix)] // NegotiateStream works on Windows and Linux, but we have not yet automated ambient credential configuration in Linux
-#else
-    [ActiveIssue(832)] // Windows Stream Security is not supported in NET Native
 #endif
     [OuterLoop]
     public static void DefaultSettings_Echo_RoundTrips_String()
     {
+#if FULLXUNIT_NOTSUPPORTED
+        bool root_Certificate_Installed = Root_Certificate_Installed();
+        bool windows_Authentication_Available = Windows_Authentication_Available();
+        if (!root_Certificate_Installed || !windows_Authentication_Available)
+        {
+            Console.WriteLine("---- Test SKIPPED --------------");
+            Console.WriteLine("Attempting to run the test in ToF, a ConditionalFact evaluated as FALSE.");
+            Console.WriteLine("Root_Certificate_Installed evaluated as {0}", root_Certificate_Installed);
```

Comment:
I wasn't evaluating if the existing conditions that were being used were correct or not, I removed it.